### ISSUE TITLE
Better documentation

### ISSUE THREAD

**Issue by drekka** [vowsjs/api-easy#33]
Really need some better documentation. I would like to see some wiki pages please with examples.

**Comment by tlianza** [comment]
I was hopeful that the links to documentation from the readme would help, but this url 404's: http://flatiron.github.com/api-easy

**Comment by kassimir** [comment]
I totally agree.  I'm using this for work, and we're behind schedule, but due to our finicky api (not written by us), other systems wouldn't correctly send POST calls.  This one does, but I've got 9 tabs open from googling how to use the damned thing.

**Comment by hboylan** [comment]
Hey guys I've made my own API test tool with a nice README if anyone's interested... 

https://www.npmjs.com/package/teqlabs-testify
